app/services/applications/change_funding_eligibility.rb

```diff
@@ -29,7 +29,11 @@ def change_funding_eligibility
       funding_eligiblity_status_code =
         (eligible_for_funding ? :marked_funded_by_policy : :marked_ineligible_by_policy)
 
-      application.update!(eligible_for_funding:, funding_eligiblity_status_code:)
+      application.assign_attributes(eligible_for_funding: eligible_for_funding, funding_eligiblity_status_code: funding_eligiblity_status_code)
+      eligible_for_funding_change = application.changes["eligible_for_funding"]
+      application.save!.tap do
+        send_eligible_for_funding_email if eligible_for_funding_change == [false, true]
```

Comment:
Can you use the `.saved_changes` to determine the change instead? seems simpler?